freelist_hmap.go

```diff
@@ -80,12 +80,21 @@ func (f *freelist) hashmapGetFreePageIDs() []common.Pgid {
 	}
 
 	m := make([]common.Pgid, 0, count)
-	for start, size := range f.forwardMap {
-		for i := 0; i < int(size); i++ {
-			m = append(m, start+common.Pgid(i))
+
+	keys := make([]common.Pgid, 0, len(f.forwardMap))
+	for k := range f.forwardMap {
+		keys = append(keys, k)
+	}
+	sort.Sort(common.Pgids(keys))
+
+	for _, start := range keys {
+		size, ok := f.forwardMap[start]
+		if ok {
+			for i := 0; i < int(size); i++ {
+				m = append(m, start+common.Pgid(i))
+			}
```

Comment:
Please consider to address the two minor comments in previous PR.

Reply:
Done.